StaticSVD.C

```diff
@@ -31,18 +33,13 @@ void dgesdd(char*, int*, int*, double*, int*,
 namespace CAROM {
 
 StaticSVD::StaticSVD(
-   StaticSVDOptions options) :
+   Options options) :
    SVD(options),
    d_samples(new SLPK_Matrix), d_factorizer(new SVDManager),
    d_this_interval_basis_current(false),
    d_max_basis_dimension(options.max_basis_dimension),
-   d_sigma_tol(options.sigma_tolerance)
+   d_singular_value_tol(options.singular_value_tol)
 {
-   CAROM_VERIFY(options.dim > 0);
```

Comment:
Why not keep these `CAROM_VERIFY` checks? For `max_basis_dimension`, the condition could be `> 0 || == -1`.

Reply:
I can add them back in. They are already checked in BasisGenerator, and only BasisGenerator can create an object of this class, so I thought it might be redundant.

Reply:
This makes sense, and I agree it is fine as it is.